config/overrides.go

```diff
@@ -361,3 +363,84 @@ func NamePrefixRemoval() tjconfig.ResourceOption {
 		r.ExternalName.OmittedFields = append(r.ExternalName.OmittedFields, "name_prefix")
 	}
 }
+
+// KnownReferencers adds referencers for fields that are known and common among
+// more than a few resources.
+func KnownReferencers() tjconfig.ResourceOption { //nolint:gocyclo
+	return func(r *tjconfig.Resource) {
+		for k, s := range r.TerraformResource.Schema {
+			if s.Computed && !s.Optional {
+				continue
+			}
+			switch {
+			case strings.HasSuffix(k, "role_arn"):
+				r.References[k] = tjconfig.Reference{
+					Type:      "github.com/crossplane-contrib/provider-jet-aws/apis/iam/v1alpha1.Role",
```

Comment:
If these type constants are used elsewhere (for example in other custom configurations), you may consider defining constants for them.

Reply:
That's a good idea but I'd like to do that for manual referencers as well altogether in another PR.